DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -481,3 +483,85 @@ def run_protocol(self, device, command, *argv, **kwarg):
 
     def parse_protocol(self, command, output, *argv, **kwarg):
         return command
+
+    def format_send_ping(self, command, *argv, **kwarg):
+        return command
+
+    def format_send_arp(self, command, *argv, **kwarg):
+        return command
+
+    @classmethod
+    def __get_ip_iface(cls, port, port_ip=None):
+        vport = cls.ixnet.Vport.find(Name=port)
+        for ip_ep in cls.ip_eps:
+            # ip -> eth -> dev group -> topo
+            topo = ip_ep.parent.parent.parent
+            if vport.href not in topo.Ports:
+                continue
+            if port_ip and port_ip not in str(ip_ep.Address):
+                continue
+            return ip_ep
+        return None
+
+    def run_send_ping(self, device, command, *argv, **kwarg):
+        """
+        - IxiaClient
+           send_ping - [port, dst_ip, src_ip]
+        """
+        params = kwarg["params"]
+        res = []
+        err = 0
+        for param in params:
+            port = param["port"]
+            dst = param["dst_ip"]
+            src = param.get("src_ip", None)
+
+            ip_ep = self.__get_ip_iface(port, src)
+            if not ip_ep:
+                continue
```

Comment:
It means that if we don't have a corresponding interface on Ixia, we will skip the ping/ARP operation. Maybe it will be better to:
`
out["port"] = port
out["src_ip"] = src
out["dst_ip"] = dst
out["arg2"] = "Failure"
out["arg3"] = "Could not find corresponding Ixia interface"
res.append(out)
continue
`